src/main/java/com/alkemy/ong/controllers/OrganizationController.java

```diff
@@ -2,31 +2,42 @@
 
 import com.alkemy.ong.dto.request.organization.OrganizationRequestDto;
 import com.alkemy.ong.dto.response.Organization.OrganizationPublicDto;
+import com.alkemy.ong.dto.response.slide.SlideResponseDto;
 import com.alkemy.ong.services.OrganizationService;
+import com.alkemy.ong.services.SlideService;
 import lombok.AllArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/organization")
 @AllArgsConstructor
 public class OrganizationController {
 
     private final OrganizationService organizationService;
+    private final SlideService slideService;
 
-    @GetMapping("/public")
+/*    @GetMapping("/public")
     public ResponseEntity<OrganizationPublicDto> publicData(@RequestParam String id) {
         OrganizationPublicDto publicDataDTO = organizationService.getPublicOrganizationData(id);
         return ResponseEntity.ok().body(publicDataDTO);
-    }
+    }*/
+@GetMapping("/public")
+public ResponseEntity<List<SlideResponseDto>> publicData(@RequestParam String id) {
+    if (organizationService.existById(id)) {
+        List<SlideResponseDto> publicDataDTO = slideService.getAllByOrganizationId(id);
```

Comment:
No vi si estaban ordenados por el campo order que pide el ticket cuando se listan, si no esta deberia implementarse.

Reply:
This is solved in the Slides repository by means of Query, SQL takes care of returning it already sorted.